models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -656,46 +662,30 @@ aeif_cond_beta_multisynapse_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = y[ S::V_M ];
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
```

Comment:
Maybe we should add a comment to assure people that we indeed want to use ``P_.V_peak_`` and not ``V_.V_peak`` here...